src/components/dashboard/components/dictionary/DictionariesSearch.jsx

```diff
@@ -19,9 +19,13 @@ export const DictionariesSearch = ({ onSearch, onSubmit, searchValue }) => (
     </div>
   </div>
 );
+DictionariesSearch.defaultProps = {
+  searchValue: '',
+};
+
 DictionariesSearch.propTypes = {
   onSearch: PropTypes.func.isRequired,
   onSubmit: PropTypes.func.isRequired,
-  searchValue: PropTypes.string.isRequired,
+  searchValue: PropTypes.string,
```

Comment:
Is this also about testing the untested areas?

Reply:
Yes, this is part of the first bullet point that handles restructuring of the code in order for it to be tested. After testing, I got warnings on the terminal console and solved it as the prop was set to required when in real sense it wasn't.